example-apps/kitchen-sink-new-router/source/components/router/start.ts

```diff
@@ -47,13 +51,18 @@ export default class Start {
 
   constructor(private router: Router) { }
 
-  gotoRouterData1() {
-    this.router.navigate(['/start/router-data1', 'Message from router']);
-  }
-
-  gotoRouterData2() {
-    this.router.navigate(['/start/router-data2',
+  goToRoute(index) {
```

Comment:
This is just an example app, but the code here seems a but confusing to me? Especially the index part of it.

Why not call the router methods directly from the template markup?